shell/platform/android/android_environment_gl.cc

```diff
@@ -24,7 +24,7 @@ AndroidEnvironmentGL::AndroidEnvironmentGL()
   if (strstr(extensions, "EGL_ANDROID_presentation_time")) {
     presentation_time_proc_ =
         reinterpret_cast<PFNEGLPRESENTATIONTIMEANDROIDPROC>(
-            eglGetProcAddress("sEGL_ANDROID_presentation_time"));
```

Comment:
should this have been `EGL_ANDROID_presentation_time` ? What is the relationship between this value and L24? @dnfield

Reply:
I think what happened here is that I _meant_ to copy and paste `eglPresentationTimeANDROID`, and instead I copy and pasted `EGL_ANDROID_presentation_time` .. and hit the s key by accident .. and then didn't notice it.

When I initially did some tests with this extension, I was just directly calling it and hacking it in as if it wouldn't have to be looked up. I then refactored to do it the right way (get the proc address this way), but never re-tested things to make sure it was still doing the good thing.. and then this got missed in review :(

Reply:
@iskakaushik FYI since I think you reviewed the original PR

Reply:
The bottom line is: this lookup is correct and we should accept it.

Reply:
Maybe can see https://cs.android.com/android/platform/superproject/+/android-10.0.0_r1:frameworks/native/opengl/libs/EGL/egl_platform_entries.cpp;l=92

Reply:
could we add a comment about the what `EGL_ANDROID_presentation_time` and `eglPresentationTimeANDROID` are used for?

It will definitely help next time there's a refactor, and the blame doesn't point to this PR. :)

Reply:
Sorry, I am not native speaker. I'm not sure my description is accurate. Maybe @dnfield could help to add this comment?

Reply:
@blasten I added some comments.